bbc/simorgh
# WSTEAM1-647 - Rename Live pages directory

Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAM1-647

Overall changes
======
- Renames the Live pages directory from `new_live` to `live`
- Updates snapshots for integration tests to reflect the directory change
- Updates any other reference of `new_live` to `live`

Testing
======
1. Pull down this branch locally
2. cd to `ws-nextjs-app` directory and run `yarn dev`
3. Navigate to http://localhost:7081/pidgin/live/c7p765ynk9qt
4. Confirm the page loads correctly
5. Navigate to http://localhost:7081/pidgin/new_live/c7p765ynk9qt
6. Confirm the page 404s correctly


Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
